cttportfolio.css

```diff
@@ -0,0 +1,941 @@
+}
+
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+	color: rgb(4, 8, 9);
+}
+
+body {
+	margin: 0;
+	background-color: rgb(2, 8, 9);
+}
+
+main>div {
+	margin: 0 auto;
+	max-width: 1300px;
+}
+
+main {
+	margin-top: 200px;
+}
+
+h4,
+h5,
+h6,
+.pp_body h1 {
+	font-weight: 700;
+	font-size: 6.0rem;
+	text-transform: uppercase;
+	-webkit-text-stroke-width: 0.3px;
+	-webkit-text-stroke-color: black;
+	color: rgb(248, 248, 255);
+}
+
+h5 {
+	font-weight: 600;
+	font-size: 4rem;
+}
+
+.am_text,
+.navlinks a,
+.contactmetext>div,
+img {
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6);
+	border-radius: 7px;
+}
+
+a.icon_hover img,
+a.pf_sub_links h6,
+.navlinks.icon_hover>a:hover {
+	border: none;
+}
+
+a.icon_hover:hover img,
+a.pf_sub_links:hover h6,
+.navlinks.icon_hover>a:hover {
+	border: 1px solid rgba(248, 248, 255, 1);
+	box-shadow: 0px -2px 5px rgba(248, 248, 255, 0.6);
+}
+
+.background_color,
+.project_text>* {
+	background-color: rgba(255, 182, 193, 0.7);
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6);
+	border-radius: 5px;
+}
+
+h1 {
+	animation: 10s fadeIn1;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+h2,
+.pp_info>div {
+	animation: 10s fadeIn2;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+nav {
+	animation: 10s fadeIn3;
+	animation-fill-mode: forwards;
+	visibility: hidden;
+}
+
+@keyframes fadeIn1 {
+	10% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	25% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+@keyframes fadeIn2 {
+	25% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	40% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+@keyframes fadeIn3 {
+	40% {
+		visibility: hidden;
+		opacity: 0;
+	}
+	65% {
+		visibility: visible;
+		opacity: 1;
+	}
+	100% {
+		visibility: visible;
+		opacity: 1;
+	}
+}
+
+/*======================================== NAV SECTION ========================================*/
+
+nav {
+	display: flex;
+	padding: 0 5%;
+	justify-content: space-between;
+	align-items: center;
+	position: fixed;
+	top: 0;
+	width: 100vw;
+	z-index: 1;
+	margin: 10px 0;
+}
+
+nav a,
+nav p {
+	text-decoration: none;
+	color: rgb(248, 248, 255);
+	text-transform: uppercase;
+}
+
+.navlogo>img {
+	padding-right: 15px;
+	height: 100%;
+	position: relative;
+	bottom: 4px;
+	box-shadow: none;
+}
+
+.navlogo>p {
+	font-size: 2.5rem;
+	font-weight: 700;
+}
+
+.navlogo {
+	display: flex;
+	align-items: center;
+	height: 80px;
+}
+
+.navlinks>a {
+	font-size: 2rem;
+	margin-left: 30px;
+	padding: 15px 10px;
+	padding-top: 18px;
+	border: none;
+	border-radius: 5px;
+	font-weight: 600;
+}
+
+.navlinks a:hover {
+	border: 1px solid rgb(248, 248, 255);
+}
+
+.nav_dropdown {
+	display: none;
+}
+
+
+/*======================================== HEADER SECTION ========================================*/
+
+header {
+	width: 100vw;
+	height: 100vh;
+	position: relative;
+	z-index: -1;
```

Comment:
negative z-index cause problems on older browsers, avoid using and give other elements positive ones instead